release.js

```diff
@@ -20,110 +20,34 @@ const childProcess = require('child_process');
 
 const inquirer = require('inquirer');
 const semver = require('semver');
-const replace = require('replace-in-file');
 const releaseit = require('release-it');
 
-const versions = require('./versions.json');
 
-const gradleFiles = [`${__dirname}/build.gradle`];
-const readmeFile = [`${__dirname}/README.md`];
-const versionsFile =  [`${__dirname}/versions.json`]
+async function versionPrompt() {
 
-async function amendGradleFiles(version) {
-  const options = {
-    files: gradleFiles,
-    from: new RegExp('BpkVersion = .+', 'g'),
-    to: `BpkVersion = '${version}'`,
-  };
-
-  try {
-    console.log('🎉  Version amended in', await replace(options));
-    return true;
-  } catch (exc) {
-    throw new Error(exc);
-  }
-}
-
-async function amendReadmeFiles(version) {
-  const options = {
-    files: readmeFile,
-    from: new RegExp('bpkVersion = ".+"', 'g'),
-    to: `bpkVersion = "${version}"`,
-  };
-
-  try {
-    console.log('🎉  Version amended in', await replace(options));
-    return true;
-  } catch (exc) {
-    throw new Error(exc);
-  }
-}
-
-async function amendVersionsFile(version) {
-  const options = {
-    files: versionsFile,
-    from: new RegExp('"backpack": ".+"', 'g'),
-    to: `"backpack": "${version}"`,
-  };
-
-  try {
-    console.log('🎉  Version amended in', await replace(options));
-    return true;
-  } catch (exc) {
-    throw new Error(exc);
-  }
-}
-
-async function versionPrompt(currentVersion) {
-  const major = semver.inc(currentVersion, 'major');
-  const minor = semver.inc(currentVersion, 'minor');
-  const patch = semver.inc(currentVersion, 'patch');
-
-  const questions = [
+  const question =
     {
-      type: 'list',
+      type: 'input',
       name: 'version',
-      message: 'What version do you want to release?',
-      choices: [
-        {
-          major,
-        },
-        {
-          minor,
-        },
-        {
-          patch,
-        },
-      ].map(i => {
-        const key = Object.keys(i)[0];
-        return {
-          key,
-          name: `${key} (${i[key]})`,
-          value: `${i[key]}`,
-        };
-      }),
-    },
-  ];
-  return await inquirer.prompt(questions);
+      message: 'What version do you want to release? (see https://github.com/Skyscanner/backpack-android/releases)',
```

Comment:
this will go away soon. in the future the version will be defined based on the github tag released (generated by release-drafter) on CI